src/components/Navbar.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import { Route, Routes, NavLink } from 'react-router-dom';
+import logo from '../images/planet.png';
+import '../styles/Navbar.css';
+import RocketsList from './navigation/RocketsList';
+import MissionsList from './navigation/MissionsList';
+import DragonsList from './navigation/DragonsList';
+import MyProfile from './navigation/MyProfile';
+
+function Navbar() {
```

Comment:
- [ ] [REQUIRED] Good job so far! After entering `"My profile"` all the performed operations(`Reservetions`) will be clean again. Please fix this error 🙏🏼 

![ezgif com-optimize](https://github.com/iamsjunaid/space-travelers-hub/assets/63915024/969f03da-91e8-4dba-911b-5e10e6356737)